reason-react-native/src/apis/PanResponder.md

```diff
@@ -1,9 +1,137 @@
 ---
-id: apis/Packager
-title: Packager
-wip: true
+id: apis/PanResponder
+title: PanResponder
```

Comment:
You can add `officialDoc: http://facebook.github.io/react-native/docs/panresponder` so a small link will be visible below huge title & will point to RN doc :)